### DISCUSSION TITLE
Windows 7 64 bit - 0.80.1 is not displaying text on screen/window (blank)

### DISCUSSION THREAD

**Discussion by turkishrational** [dosbox-staging/dosbox-staging#3032] [General]
DOSBOX STAGING is running (teher is a keyboard response and recognizes command) but can not display anything on its screen/window (blabk/black windows) Operating System: Windows 7 64 bit.
(I have tried it by extracting to a folder, without using setup.exe.. zip: dosbox-staging-windows-x86_64-v0.80.1

**Comment by kcgen** [comment]
Open a bug report and fill out all the fields.

**Comment by MasterO2** [comment]
@turkishrational, to open a bug report, go to https://github.com/dosbox-staging/dosbox-staging/issues, and click on `New Issue`.
